packages/data-context/src/sources/CloudDataSource.ts

```diff
@@ -174,6 +152,18 @@ export class CloudDataSource {
     return `${print(config.document)}-${stringifyVariables(config.variables)}`
   }
 
+  #formatWithErrors (data: OperationResult<any, any>) {
+    // If we receive a 401 from the dashboard, we need to logout the user
```

Comment:
```suggestion
  #formatWithErrors: (data: OperationResult<any, any>) => {
    // If we receive a 401 from the dashboard, we need to logout the user
```

Needs to be an arrow fn to reference `this` correctly the way we're passing it to `then`